### PR TITLE
LICENSE.txt: Add missing closing parentheses

### PR DESCRIPTION
**Pull request by wjt**
I have read `CONTRIBUTING.md`:

> The Steam Link app is not open-source and does not accept external contributions.

I have also read the manifest and can see that this will cause the build to fail unless the corresponding file in the binary distribution is updated.

Nonetheless, a PR seems the easiest way to report the typos I've spotted.

### CODE REVIEW

**Comment by smcv** [comment]
Thanks! Because the version in the binary code-drop is canonical, this will have to happen there first.

File: LICENSE.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ STEAM LINK LICENSE (FLATHUB)
...
+The software application(s) (the "Program") is the copyrighted work of Valve Corporation ("Valve") or its suppliers.  All rights reserved, except as expressly stated herein.  Valve has granted Flathub the right to distribute the Program, including via its website https://flathub.org/.  The Program is provided solely for installation by end users according to the terms of this License Agreement. All use of the Program is governed by the terms of the Steam subscriber agreement located at www.steampowered.com/agreement (the "Steam Agreement"), as such terms may be updated from time to time, which terms are incorporated into this License Agreement by this reference. Any use, reproduction or redistribution of the Program not in accordance with the terms of the License Agreement and the Steam Agreement is expressly prohibited.
```
Comment (smcv) [inline LICENSE.txt:5]: @slouken, it looks like you fixed the first of these two (after `"Program"`), but not the second (after `"Valve"`).

**Comment by smcv** [comment]
This change is not going to build successfully unless/until there is a binary code-drop from Valve with the corresponding change. @slouken, please could you queue that internally for the next release?

**Comment by slouken** [comment]
Done!

**Pull request closed without merging**